ogre2/src/Ogre2ParticleNoiseListener.hh

```diff
@@ -51,8 +51,13 @@ namespace ignition
       private: Ogre::MaterialPtr ogreMaterial;
 
       /// \brief Particle scatter ratio. This is used to determine the ratio of
-      /// particles that will detected by sensors
-      private: double particleScatterRatio = 0.65;
+      /// particles that will detected by sensors. Increasing the ratio
```

Comment:
```suggestion
      /// particles that will be detected by sensors. Increasing the ratio
```

Reply:
resolved in https://github.com/ignitionrobotics/ign-rendering/pull/269/commits/0c48d1685d691d45b55f248acfd4813b0fe1f350